src/sql/workbench/contrib/welcome/gettingStarted/browser/initialSetupWizard.css

```diff
@@ -0,0 +1,280 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.ads-initial-setup-wizard {
```

Comment:
are all these css classes really necessary? did you handcraft them or copy it from somewhere?

Reply:
@alanrenmsft  Are you concerned about the length of the CSS class? This will have very minimal impact on performance. These classes are descriptive and effectively scope all styles to the initial setup wizard.